### PR TITLE
49 set row length to standard browser window

### PR DESCRIPTION
**Pull request by janiswehen**

### CODE REVIEW

File: packages/PoppyPrint-Core.package/PPFormatterConfig.class/instance/maxLineWidth..st [hunk 1]
```diff
@@ -0,0 +1,4 @@
+accessing
+maxLineWidth: anObject
```
Comment (Yannis-Hofmann) [inline packages/PoppyPrint-Core.package/PPFormatterConfig.class/instance/maxLineWidth..st:2]: anInteger vs anObject

File: packages/PoppyPrint-Core.package/PPFormatterConfig.class/instance/formatToMaxLineWidth..st [hunk 2]
```diff
@@ -0,0 +1,4 @@
+accessing
+formatToMaxLineWidth: anObject
```
Comment (Yannis-Hofmann) [inline packages/PoppyPrint-Core.package/PPFormatterConfig.class/instance/formatToMaxLineWidth..st:2]: aBoolean vs anObject

File: packages/PoppyPrint-Tests.package/PPWondowFormatterTest.class/properties.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+	"category" : "PoppyPrint-Tests",
+	"classinstvars" : [
+		 ],
+	"classvars" : [
+		 ],
+	"commentStamp" : "",
+	"instvars" : [
+		 ],
+	"name" : "PPWondowFormatterTest",
```
Comment (Yannis-Hofmann) [inline packages/PoppyPrint-Tests.package/PPWondowFormatterTest.class/properties.json:10]: This should be PPWindowFormatterTest

**Review by Yannis-Hofmann — CHANGES_REQUESTED** [review]

File: packages/PoppyPrint-Tests.package/PPFormatterTest.class/instance/testStandaredWindowWidth1.st [hunk 4]
```diff
@@ -0,0 +1,13 @@
+tests - WindowWidth
+testStandaredWindowWidth1
```
Comment (kdauer) [inline packages/PoppyPrint-Tests.package/PPFormatterTest.class/instance/testStandaredWindowWidth1.st:2]: Should be Standard or could be called Default, maybe Default is a better naming convention

File: packages/PoppyPrint-Tests.package/PPFormatterTest.class/instance/testStandaredWindowWidth2.st [hunk 5]
```diff
@@ -0,0 +1,13 @@
+tests - WindowWidth
+testStandaredWindowWidth2
```
Comment (kdauer) [inline packages/PoppyPrint-Tests.package/PPFormatterTest.class/instance/testStandaredWindowWidth2.st:2]: Also Standard of Default

File: packages/PoppyPrint-Tests.package/PPFormatterTest.class/instance/testStandaredWindowWidth3.st [hunk 6]
```diff
@@ -0,0 +1,12 @@
+tests - WindowWidth
+testStandaredWindowWidth3
```
Comment (kdauer) [inline packages/PoppyPrint-Tests.package/PPFormatterTest.class/instance/testStandaredWindowWidth3.st:2]: Standard or Default

**Review by kdauer — CHANGES_REQUESTED** [review]

File: packages/PoppyPrint-Core.package/PPWindowFormatter.class/instance/putIndentedNewlineAndWord.st [hunk 7]
```diff
@@ -0,0 +1,8 @@
+put
+putIndentedNewlineAndWord
+
+	resultStream nextPut: Character cr.
+	resultStream next: tabCount + 1 put: Character tab.
+	resultStream nextPutAll: wordStream contents.
+	lineWidth := tabCount + 1 + wordWidth.
+	index := index + wordWidth + 1.
```
Comment (kdauer) [inline packages/PoppyPrint-Core.package/PPWindowFormatter.class/instance/putIndentedNewlineAndWord.st:8]: Is this a trailing dot?

File: packages/PoppyPrint-Core.package/PPWindowFormatter.class/instance/putSeperatorAndWord.st [hunk 8]
```diff
@@ -0,0 +1,7 @@
+put
+putSeperatorAndWord
+
+	resultStream nextPut: self charAtIndex.
+	resultStream nextPutAll: wordStream contents.
+	lineWidth := lineWidth + wordWidth + 1.
+	index := index + wordWidth + 1
```
Comment (kdauer) [inline packages/PoppyPrint-Core.package/PPWindowFormatter.class/instance/putSeperatorAndWord.st:7]: Or is the dot here missing?

File: packages/PoppyPrint-Tests.package/PPWindowFormatterTest.class/instance/testFormatingToDefaultWindowWidth3.st [hunk 9]
```diff
@@ -0,0 +1,10 @@
+tests
+testFormatingToDefaultWindowWidth3
```
Comment (kdauer) [inline packages/PoppyPrint-Tests.package/PPWindowFormatterTest.class/instance/testFormatingToDefaultWindowWidth3.st:2]: testFormattingToDefaultWindowWidth3, Formatting

File: packages/PoppyPrint-Tests.package/PPWindowFormatterTest.class/instance/testFormatingToDefaultWindowWidth1.st [hunk 10]
```diff
@@ -0,0 +1,13 @@
+tests
+testFormatingToDefaultWindowWidth1
```
Comment (kdauer) [inline packages/PoppyPrint-Tests.package/PPWindowFormatterTest.class/instance/testFormatingToDefaultWindowWidth1.st:2]: Formatting

File: packages/PoppyPrint-Tests.package/PPWindowFormatterTest.class/instance/testFormatingToDefaultWindowWidth2.st [hunk 11]
```diff
@@ -0,0 +1,9 @@
+tests
+testFormatingToDefaultWindowWidth2
```
Comment (kdauer) [inline packages/PoppyPrint-Tests.package/PPWindowFormatterTest.class/instance/testFormatingToDefaultWindowWidth2.st:2]: Formatting

File: packages/PoppyPrint-Tests.package/PPWindowFormatterTest.class/instance/testFormatingToZeroWidth1.st [hunk 12]
```diff
@@ -0,0 +1,20 @@
+tests
+testFormatingToZeroWidth1
```
Comment (kdauer) [inline packages/PoppyPrint-Tests.package/PPWindowFormatterTest.class/instance/testFormatingToZeroWidth1.st:2]: Formatting

**Review by kdauer — CHANGES_REQUESTED** [review]

**Review by kdauer — APPROVED** [review]

**Pull request merged**